public/js/dashboardInfoForm.js

```diff
@@ -1,3 +1,5 @@
+/* global google*/
```

Comment:
nit: remove?

Reply:
This line is required by eslint to prevent it from throwing a `google is not defined` error.